notification-server/resources/schema.sql

```diff
@@ -0,0 +1,6 @@
+-- accounts created.
+CREATE TABLE IF NOT EXISTS account_device_mapping (
+  id SERIAL8 PRIMARY KEY,
+  address BYTEA NOT NULL UNIQUE,
+  device_id VARCHAR NOT NULL UNIQUE
```

Comment:
This is not correct is it? As I see it, the unique restriction should be on the combination of `address` and `device_id`:
```suggestion
  address BYTEA NOT NULL,
  device_id VARCHAR NOT NULL,
  UNIQUE (address, device_id)
```

Reply:
One could argue on setting UNIQUE on device_id instead,  but address should definitely not be UNIQUE.